bigchaindb/commands/election_types.py

```diff
@@ -16,5 +16,14 @@
                 'help': 'Path to the private key of the election initiator.'
             }
         }
+    },
+    'migration': {
```

Comment:
This particular migration is connected to the ABCI (Tendermint) chain upgrades. We might introduce other migrations in the future. So I would call it `tendermint-chain-migration` or `abci-chain-migration`.

Reply:
The keys from this `dict` directly translate into the CLI methods, as in `election new tendermint-chain-migration`. I'd prefer to keep them short if possible, for user convenience. How about `chain-migration` as a compromise?

Reply:
The election transaction `operation` would also need to be changed to `CHAIN_MIGRATION_ELECTION`

Reply:
Done. I also refactored the Class name etc.